tsconfig.json

```diff
@@ -1,20 +1,11 @@
 {
-  "compilerOptions": {
-    "target": "es5",
-    "lib": ["dom", "dom.iterable", "esnext"],
-    "allowJs": true,
-    "skipLibCheck": true,
-    "esModuleInterop": true,
-    "allowSyntheticDefaultImports": true,
-    "strict": true,
-    "forceConsistentCasingInFileNames": true,
-    "noFallthroughCasesInSwitch": true,
-    "module": "esnext",
-    "moduleResolution": "node",
-    "resolveJsonModule": true,
-    "isolatedModules": true,
-    "noEmit": true,
-    "jsx": "react-jsx"
-  },
-  "include": ["src"]
+  "files": [],
+  "references": [
+    {
+      "path": "./tsconfig.app.json"
+    },
+    {
+      "path": "./tsconfig.node.json"
```

Comment:
Do we need this?

Reply:
No, not at all, is just some more noise added by `create-vite` which we don't need for this purposes so I've removed it in [eb4c818](https://github.com/primait/interview-fe-public/pull/8/commits/eb4c818feb32675ac635f2b62655c3094371fa03), thanks!